QUEST.tsv

```diff
@@ -1407,96 +1408,96 @@ QUEST_20150317_001407	Sincerity of Moze
+QUEST_20150323_001435	It could be a broken plaster of the Goddess Statue so get rid of the monsters there. {nl}If I'm right, then Naktis must be behind it. {nl}I may be far away but I hear the stories.
+QUEST_20150323_001436	Naktis is looking for a clue to enter the Cathedral. {nl}If I'm right, not only is the Forest of Prayer like that but all of Pilgrim Path. {nl}If you reach the Cathedral, please defeat Naktis. {nl}You have to stop the revelation from falling into demon hands.
+QUEST_20150323_001437	This place is dying. {nl}All the Tree Root Crystals have mutated.
+QUEST_20150323_001438	Some Tree Root Crystals are useless unless burned. {nl}Some even fume at the core.
+QUEST_20150323_001439	We must do whatever it takes to turn it back but… {nl}It's all so tiring now…
+QUEST_20150323_001440	I feel like I'm drowning. {nl}The Tree Root Crystal is slowly killing me.
+QUEST_20150323_001441	Do whatever it takes to destroy it… {nl}Please save me.
+QUEST_20150323_001442	I've lost all desire. Am I dying now? {nl}Eating and even breathing feels meaningless.
+QUEST_20150323_001443	But it must go on. {nl}Unless that cursed Tree Root Crystal is destroyed…
+QUEST_20150323_001444	There are things that Witas care about. {nl}Won't he get his mind back if you show him those? 
 QUEST_20150323_001445	Somebody's Diary
-QUEST_20150323_001446	Those brothers are crazy. They must be completely nuts. {nl}When did it start? When Dora attacked Gracious?{nl}When they saw the demons marching? 
-QUEST_20150323_001447	It became worse since Gracius was left behind due to injury.{nl}I zipped my mouth and everyone's eyes turned murderous. {nl}It was like a volcano was about to explode. {nl}I thought it was a lesson until then.
-QUEST_20150323_001448	Then the trigger was released. {nl}A brother I can't remember had smacked down Dion with a mace when he was sleeping and I…{nl}Everyone…{nl}No one could see their old selves desiring to become a priest. 
-QUEST_20150323_001449	This surely is not a teaching of sir Mayvern.{nl}Greed? Laziness? How can you call killing each other a lesson? {nl}I even envy Dion for dying first. {nl}
-QUEST_20150323_001450	I can hear snickering somewhere. {nl}It could be the sound of Dora deceiving me. {nl}It doesn't really matter anymore. 
-QUEST_20150323_001451	Trapped soul gave the scripture and disappeared.
-QUEST_20150323_001452	Thanks to the medicine, Witas' curse is not working.
-QUEST_20150323_001453	If you ever see the Devout Tree, please bring it back to life. {nl}I heard that tree blocks the power of the curse.
+QUEST_20150323_001446	Those brothers are crazy. They must be completely nuts. {nl}When did it start? When Dora attacked Gracius? {nl}When they saw the demons marching?
+QUEST_20150323_001447	It became worse since Gracius was left behind injured. {nl}I kept my mouth shut while everyone's eyes turned murderous. {nl}The town was like a volcano waiting to erupt. {nl}I thought it was a lesson until then.
```

Comment:
While it's true that Gracius was left behind while injured, changing the sentence in this way doesn't convey the same thing.  He was left behind because he was injured, one thing caused the other.  Leaving it like this makes it sound more heartless, they just left him behind injured.

I am unsure where "the town" comes from?  It's also possible that this means that the situation was ( tense ) like a volcano waiting to erupt.

I'd guess that this line could use some translation love as well.